.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+## Description
+
+TODO: Link to a JIRA ticket if one has been created for this feature/issue.
+Use the following format for a link: `[FEDX-100](https://openedx.atlassian.net/browse/FEDX-100)`
+
+TODO: Include a detailed description of the changes in the body of the PR.
+
+TODO: If relevant, provide a URL to updated Pattern Library documentation.
+
+## Manual Test Plan
+
+TODO: If your changes require manual tests, include instructions for any required manual tests, and any manual testing that has already been performed.
+
+## Testing Checklist
+- [ ] Write unit tests for all new features.
+- [ ] Write accessibility (a11y) tests for all new UI.
+- [ ] Manually test responsive behavior.
+
+## Non-testing Checklist
+- [ ] Consider any documentation your change might need, and which users will be affected by this change.
+- [ ] Consult with the Pattern Library team if adding a new component.
+
+## Post-review
+- [ ] Squash commits into discrete sets of changes with descriptive commit messages.
+
+## Reviewers
```

Comment:
The mention bot is connected to this repo now (as is the openedx-webook bot - not sure what that one does). I forgot to remove this section, so thanks for reminding me :)